src/utils/minioClient.ts

```diff
@@ -386,7 +386,14 @@ export async function minioListenerHandler(bucket: string, file: string, tableNa
  */
 export async function downloadFileAndUpload(bucket: string | undefined) {
```

Comment:
We need to factor in the functionality of obtaining the bucket variable value and appending it to the buckets array.